### ISSUE TITLE
downloadURL : undefinedhttp

### ISSUE THREAD

**Issue by bguedel** [electrode-io/electrode-ota-server#6]
while trying to get the files from the file server, i have my setup such as

```js 
    "electrode-ota-server-fileservice":{
              "options": {
                "downloadUrl": "http://my.url.com/storagev2"
              }
        },
```

however my payload is showing up as such

```js
"downloadURL": "undefinedhttp://my.url.com/storagev2/superlonghashthatdoesntreallymatter",
```

I believe the issue exists [here](https://github.com/electrode-io/electrode-ota-server/blob/2a1bc952aa41c696ae24b1a5f7f4485ee96f41c3/server/fileservice/dao.js#L12) with an extra set of parens

here is the POC
https://jsfiddle.net/hLdL4dn5/

**Comment by taina0407** [comment]
Fixed and waiting for merge in #8

**Comment by sdg9** [comment]
FYI #8 is merged this issue can be closed.
